loops.js

```diff
@@ -51,3 +66,12 @@
 // >> 9
 // ==========================================
 
+for (let i = 0; i <= 9; i++) {
+  if (i <= 2) {
+    console.log(`${i}`);
+  } else if (i <= 5) {
+    console.log(`> ${i}`);
+  } else {
+    console.log(`>> ${i}`);
+  }
+}
```

Comment:
Werkt perfect! In de uitwerkingen is het op een iets andere manier gedaan maar ik vind jouw manier ook beter!

Dit staat in de uitwerkingen:

> for (let i = 0; i < 10; i++) {
>   if (i < 3) {
>     console.log(i);
>   }
>   if (i >= 3 && i < 6) {
>     console.log('> ' + i);
>   }
>   if (i >= 6) {
>     console.log('>> ' + i);
>   }
> }